aptos-move/framework/aptos-framework/sources/object_code_deployment.move

```diff
@@ -75,6 +78,21 @@ module aptos_framework::object_code_deployment {
         object_address: address,
     }
 
+    struct ObjectCodePermission has copy, drop, store {}
```

Comment:
does this need a separate permission from code.move? I am not sure I understand the difference between the two files fully.

Reply:
Done